tests/integration/test_model_steps.py

```diff
@@ -48,6 +50,12 @@ def test_model_existence_check():
     status, out = run_scenario('leaves', 'existence', 1)
     assert_equals(status, 0, out)
 
+    # One of the gardens has a non-ASCII name. On Python 2, it gets output by
+    # Nose using the escapes
+    garden4 = '颐和园'
+    if not PY3:
+        garden4 = repr(garden4)[2:-1]
```

Comment:
I don't even remotely understand what this is doing. It looks suss to me though.